### DISCUSSION TITLE
Bug With messages...

### DISCUSSION THREAD

**Discussion by HydroZenon** [ppy/osu#17311] [Q&A] [answered]
[performance.log](https://github.com/ppy/osu/files/8299675/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8299677/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8299678/updater.log)
[database.log](https://github.com/ppy/osu/files/8299679/database.log)
[network.log](https://github.com/ppy/osu/files/8299680/network.log)
https://streamable.com/t5v04m here's the clip.

**Comment by HydroZenon** [comment]
after this it crashes. I thought it was resolved. Idk if im doing something wrong on my end or what.

**Comment by peppy** [comment, marked as answer]
Already reported, already fixed. Please wait for the next release and try using search next time!
